cas-server-core-api-authentication/src/main/java/org/jasig/cas/authentication/RootCasException.java

```diff
@@ -12,7 +12,7 @@
  * see org.jasig.cas.authentication.handler.AuthenticationException
  * @since 4.0.0
  */
-public abstract class RootCasException extends Exception {
+public abstract class RootCasException extends RuntimeException {
```

Comment:
There is nothing here that would disallow cascades obviously. There is just no benefit to explicitly declare that, I don't think.